app/jobs/track_agent_statuses_job.rb

```diff
@@ -4,6 +4,7 @@ class TrackAgentStatusesJob
 
   def perform
     current = BackendService.new.get_agent_online_state.map do |data|
+      agent_has_eaten(data) if data[:status] == 'Ruokatunti'
```

Comment:
There's also seems to be a tiny amount of statuses that contains substrings like "lounas", "ruoka" and "safka" but that's only 36 compared to 5146 normal cases in the old data.